test/hisi_zip_test/test_lib.h

```diff
@@ -111,7 +113,13 @@ struct hizip_test_info {
 };
 
 void *send_thread_func(void *arg);
-int create_threads(struct hizip_test_info *info);
+void *poll_thread_func(void *arg);
+int create_send_threads(struct hizip_test_info *info,
+			void *(*send_thread_func)(void *arg),
+			int num);
+int create_poll_threads(struct hizip_test_info *info,
+			void *(*poll_thread_func)(void *arg),
+			int num);
```

Comment:
And logically, there should have the "free" functions about create_send_threads and create_poll_threads